metpy/calc/thermo.py

```diff
@@ -487,7 +487,7 @@ def dewpoint_rh(temperature, rh):
     temperature : `pint.Quantity`
         Air temperature
     rh : `pint.Quantity`
-        Relative humidity expressed as a ratio in the range (0, 1]
+        Relative humidity expressed as a ratio in the range 0 <= rh <= 1
```

Comment:
Forgot to mention:

I changed the formatting of this line, because having unclosed brackets messes up the syntax highlighting in most text editors. Having said that, I think in order for `(0,1]` being equivalent to my expression it should probably read `0 < rh <= 1?

Reply:
Fair enough on the change--yes, it should be your latter expression.